handlers/soft-opt-in-consent-setter/cfn.yaml

```
@@ -113,6 +117,117 @@ Resources:
               Resource:
                 - arn:aws:s3::*:membership-dist/*
 
+  LambdaFunctionIAP:
+    Type: AWS::Serverless::Function
+    Condition: IsDev
+    Properties:
+      Description: Updates Identity Soft Opt-In Consents upon Acquisition and Cancellation of Subscriptions in Salesforce
+      FunctionName: !Sub soft-opt-in-consent-setter-IAP-${Stage}
+      Handler: com.gu.soft_opt_in_consent_setter.HandlerIAP::handleRequest
+      CodeUri:
+        Bucket: support-service-lambdas-dist
+        Key: !Sub membership/${Stage}/soft-opt-in-consent-setter/soft-opt-in-consent-setter.jar
+      MemorySize: 512
+      Runtime: java11
+      Environment:
+        Variables:
+          Stage: !Ref Stage
+          sfApiVersion: v46.0
```

Could we update this to the latest version pleeease? It would nice to move this into config or maybe in secrets manager alongside credentials for all lambdas at some point.
```suggestion
          sfApiVersion: v56.0
```